catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -21,14 +21,19 @@
             name="Directory Path" id="monitoredDirectoryPath" required="true"
             type="String" default=""/>
 
-        <AD description="Optional: Copy the ingested files into a backup directory under the monitored directory named /.ingested  -  NOTE: this will double the amount of disk space required for ingested files in this monitored directory if its Processing Directive includes storing the file in the DDF Content Repository."
-            name="Copy Files to Backup Directory" id="copyIngestedFiles" required="false"
-            type="Boolean" default="false"/>
+        <AD description="Choose what happens to the content item after it is ingested. Delete will remove the original file after storing it in the content store. Move will store the item in the content store, and a copy under ./ingested, then remove the original file. NOTE: this will double the amount of disk space used. Monitor in place will index the file and serve it from its original location."
```

Comment:
I would suggest putting the whole `NOTE: ...` in parentheses to help separate it from the Monitor in place descriptions that could appear to be part of the note for Move.